docs/GlobalAPI.md

```diff
@@ -731,6 +731,60 @@ test.each`
 });
```

Comment:
oh, and `test.concurrent.failing` should work

Reply:
Ok. I think I've managed to cover this! Though I'm not sure how to document it because it will be at least another few cases in README. `test.concurrent.failing`, `test.concurrent.skip.failing`, `test.concurrent.only.failing`.

Reply:
Should I add all of those? If so do we want it to say like `Works like test.failing but concurrently`? Or a whole copy of description?